persistence/src/test/scala/hmda/persistence/institutions/InstitutionPersistenceSpec.scala

```diff
@@ -34,6 +34,24 @@ class InstitutionPersistenceSpec extends ActorSpec {
       probe.send(institutionsActor, GetInstitutionById(modified.id.toString))
       probe.expectMsg(modified)
     }
+
+    "return a set of institutions matching a list of ids" in {
+      // Setup: Persist demo institutions
+      val i1 = Institution(71, "CFPBank", Set(ExternalId("99-1234567", FederalTaxId)), CFPB, Bank, hasParent = true, Active)
+      val i2 = Institution(72, "FRBank", Set(ExternalId("654321", RssdId)), CFPB, Bank, hasParent = true, Active)
+      val i3 = Institution(73, "MLBank", Set(ExternalId("externalTest0", FdicCertNo)), FDIC, Bank, hasParent = true, Active)
+      for (institution <- List(i1, i2, i3)) {
+        probe.send(institutionsActor, CreateInstitution(institution))
```

Comment:
You could always convert it to a list with `DemoData.testInstitutions.toList` or another type that is easier to work with :)!